src/Directory.Build.targets

```diff
@@ -2,9 +2,9 @@
   <Import Project="..\Directory.Build.targets" Condition="Exists('..\Directory.Build.targets')" />
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*">
+    <!-- <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*">
```

Comment:
Perhaps get sourcelink back in or delete it all together if it won't be possible?

Reply:
Good catch. I didn't mean to actually check this in. I figured out the issue and it's no longer an issue.